manifests/quick-start/base/overlays/argo-server-deployment.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: argo-server
+spec:
+  template:
+    spec:
+      containers:
+        - name: argo-server
+          args:
+            - server
+            - --namespaced
+            - --auth-mode
```

Comment:
* [ ] Check to make sure `--auth-mode sso` does not result in both `server` and `sso`.